fuzzywuzzy/fuzz.py

```diff
@@ -159,17 +157,10 @@ def _token_set(s1, s2, partial=True, force_ascii=True, full_process=True):
     diff1to2 = tokens1.difference(tokens2)
     diff2to1 = tokens2.difference(tokens1)
 
-    sorted_sect = " ".join(sorted(intersection))
-    sorted_1to2 = " ".join(sorted(diff1to2))
-    sorted_2to1 = " ".join(sorted(diff2to1))
-
-    combined_1to2 = sorted_sect + " " + sorted_1to2
-    combined_2to1 = sorted_sect + " " + sorted_2to1
-
-    # strip
-    sorted_sect = sorted_sect.strip()
-    combined_1to2 = combined_1to2.strip()
-    combined_2to1 = combined_2to1.strip()
+    # sort, join, and strip
+    sorted_sect = (" ".join(sorted(intersection))).strip()
+    combined_1to2 = (" ".join([sorted_sect, " ".join(sorted(diff1to2))])).strip()
+    combined_2to1 = " ".join([sorted_sect, " ".join(sorted(diff2to1))]).strip()
```

Comment:
This is pretty much the same, but not completely. In this changed code whitespace from `sorted_sect` is stripped before the strings are combined, while it was done afterwards in the original code. This does not matter in this case, since the original strip for `sorted_sect` was not required, since `" ".join(sorted(intersection))` can never have whitespaces in begin or end. From a performance standpoint this does not really matter, but is probably slightly slower, since it creates some intermediate lists and calls the join function more often.